app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  around_action :unscope_resources
+  skip_before_filter :authenticate_user!
+  load_resource :user
+  skip_authorize_resource :user
+
+  def show
+    @course_users = @user.course_users.from_instance(current_tenant)
```

Comment:
Need to add the calculated attributes here - achievement_count.

Calculating current_level also requires the calculation of `experience_points` Can you check to see if adding the `experience_points` here would reduce the number of SQL calls? Can check the dev logs for this.